.github/workflows/deps_lint.yml

```diff
@@ -87,6 +87,31 @@ jobs:
         .ci/openapi_diff.sh
         .ci/index_openapi_diff.sh
 
+  docker-image:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Build the Docker image
+      run: docker-compose build
+
+    - name: Test server
+      run: |
+        docker-compose up optimade &
+        .github/workflows/wait_for_it.sh localhost:3213 -t 120
+        sleep 15
+        curl http://localhost:3213/optimade/info > info.json
+        grep -F "http://example.com" info.json || exit 1
+
+    - name: Test index server
+      run: |
+        docker-compose up optimade-index &
+        .github/workflows/wait_for_it.sh localhost:3214 -t 120
+        sleep 15
+        curl http://localhost:3214/optimade/info > info.json
+        grep -F "http://example.com" info.json || exit 1
```

Comment:
Likewise here:

```suggestion
        docker exec optimade-python-tools_optimade-index_1 optimade_validator http://localhost:5001/optimade/v0 --index
```

Reply:
This is working _excellently_!! Well done with this validator!